x-pack/solutions/security/plugins/security_solution/public/configurations/page/ai_settings_route_guard.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+import { Redirect } from 'react-router-dom';
+import { AISettings } from '../tabs/ai_settings';
+import { useAgentBuilderAvailability } from '../../agent_builder/hooks/use_agent_builder_availability';
+import { useNavigation } from '../../common/lib/kibana';
+import { SecurityPageName } from '@kbn/security-solution-navigation';
+import { CONFIGURATIONS_PATH } from '../../../common/constants';
+import { ConfigurationTabs } from '../constants';
+
+export const AISettingsRouteGuard: React.FC = React.memo(() => {
```

Comment:
@spong This in addition to changes [in this commit](https://github.com/stephmilovic/kibana/commit/00a841c32892c57e2cbe6dccf4e25171a892d230) prevent AI Settings from showing in EASE

<img width="1192" height="607" alt="Screenshot 2025-12-12 at 12 07 35 PM" src="https://github.com/user-attachments/assets/68dacbf8-e1dc-4acf-8894-8a46a4e33aa5" />